.github/workflows/all_solutions.yml

```diff
@@ -26,19 +33,24 @@ env:
   DOTNET_NOLOGO: true
 
 jobs:
-  build-profiler:
-    uses: ./.github/workflows/build_profiler.yml
-    secrets: inherit
-    permissions:
-      pull-requests: read
-      contents: write
-      packages: write
+
+  # build-profiler:
```

Comment:
Is this work for this PR, or preparation for other changes?

Reply:
Commented out but not removed, just in case we come back to wanting to execute the build_profiler workflow from the all_solutions workflow. Call it indecision on my part, but I want to leave it commented out for now... ;-)